Backpack/Calendar/Classes/BPKCalendar.m

```diff
@@ -1,5 +1,155 @@
+    
+    BPKCalendarAppearance *appearance = [BPKCalendarAppearance fromFSCalendarAppearance:self.calendarView.appearance];
+    appearance.headerDateFormat = @"MMMM";
+    appearance.headerTitleColor = [BPKColor gray900];
+    appearance.separators = FSCalendarSeparatorNone;
+    appearance.weekdayFont = weekdayTextAttributes[NSFontAttributeName];
+    appearance.weekdayTextColor = weekdayTextAttributes[NSForegroundColorAttributeName];
+    appearance.todayColor = [BPKColor gray100];
+    appearance.titleTodayColor = [BPKColor gray900];
+    appearance.titleDefaultColor = [BPKColor gray900];
+    appearance.selectionColor = [BPKColor blue500];
+    appearance.headerTitleFontStyle = BPKFontStyleTextLgEmphasized;
+    
+    Ivar ivar = class_getInstanceVariable(FSCalendar.class, "_appearance");
+    object_setIvar(self.calendarView, ivar, appearance);
+    
+    // Register our Custom Header Class
+    [self.calendarView.calendarHeaderView.collectionView registerClass:[BPKCalendarHeaderCell class] forCellWithReuseIdentifier:@"cell"];
+    [self.calendarView.collectionView registerClass:[BPKCalendarStickyHeader class] forSupplementaryViewOfKind:UICollectionElementKindSectionHeader withReuseIdentifier:@"header"];
+    
+    [self addSubview:self.calendarView];
+    
+    [NSLayoutConstraint activateConstraints:@[
+                                              [self.calendarView.topAnchor constraintEqualToAnchor:self.topAnchor],
+                                              [self.calendarView.bottomAnchor constraintEqualToAnchor:self.bottomAnchor],
+                                              [self.calendarView.leadingAnchor constraintEqualToAnchor:self.leadingAnchor],
+                                              [self.calendarView.trailingAnchor constraintEqualToAnchor:self.trailingAnchor]
+                                              ]];
+    
+    _calendarWeekdayView = [[FSCalendarWeekdayView alloc] initWithFrame:CGRectZero];
+    _calendarWeekdayView.translatesAutoresizingMaskIntoConstraints = NO;
+    _calendarWeekdayView.calendar = _calendarView;
+    _calendarWeekdayView.backgroundColor = [BPKColor white];
+    [self addSubview:_calendarWeekdayView];
+    
+    UIView *bottomBorder = [[UIView alloc] initWithFrame:CGRectZero];
+    bottomBorder.translatesAutoresizingMaskIntoConstraints = NO;
+    bottomBorder.backgroundColor = [BPKColor gray100];
+    [_calendarWeekdayView addSubview:bottomBorder];
+    
+    [NSLayoutConstraint activateConstraints:@[
+                                              [_calendarWeekdayView.topAnchor constraintEqualToAnchor:self.topAnchor],
+                                              [_calendarWeekdayView.leadingAnchor constraintEqualToAnchor:self.leadingAnchor],
+                                              [_calendarWeekdayView.trailingAnchor constraintEqualToAnchor:self.trailingAnchor],
+                                              [_calendarWeekdayView.heightAnchor constraintEqualToConstant:48]
```

Comment:
Can we make the constant 6 times `BPKSpacingMd` instead?

Reply:
done